File: Controls/EditorA/utillibrary/src/main/java/com/kelfan/utillibrary/NotificationWorker.java
package com.kelfan.utillibrary;

import android.content.Context;
import android.widget.Toast;

/**
 * Created by Administrator on 1/01/2018.
 */

public class NotificationWorker {
    public static void notifyResult(Context context, String action,int result) {
        if (result == 1) {
            Toast.makeText(context, action + " success", Toast.LENGTH_SHORT).show();
        } else {
            Toast.makeText(context, action + " fail", Toast.LENGTH_SHORT).show();
        }
    }
}

File: MyProject/07KotlinEditor/app/src/main/java/com/kelfan/kotlineditor/util/NotificationWorker.kt
package com.kelfan.kotlineditor.util

import android.content.Context
import android.widget.Toast

/**
 * Created by Administrator on 21/01/2018.
 */
class NotificationWorker {
    companion object{
        fun notifyResult(context: Context, action: String, result: Int) {
            if (result == 1) {
                Toast.makeText(context, action + " success", Toast.LENGTH_SHORT).show()
            } else {
                Toast.makeText(context, action + " fail", Toast.LENGTH_SHORT).show()
            }
        }
    }
}